Again adding some more of my servers

Fusl opened the pull request.

jedisct1 commented:
Hi!

`fvz-rec-de-dus-01`'s fingerprint is interesting :)

Other resolvers that currently don't work:

```
- fvz-rec-fr-rbx-01 - Failed
- fvz-rec-fr-rbx-01 - Failed
- fvz-rec-pl-mz-01 - Failed
- fvz-rec-ru-mow-01 - Failed
```

Fusl commented:
Hi,

thanks for letting me know that the fingerprint is broken. That damn stupid deployment software I'm using is not doing a good job :( I fixed this in #263.

The provider stopped hosting fvz-rec-fr-rbx-01 and will stop hosting the other one in France too so I also removed that.

fvz-rec-pl-mz-01 That provider is blocking UDP on port 443. I wrote them an email now, lets see if they removed it. If not, there are still plenty of other ports the daemon is listening on (54, 5353, 27015, ...).

fvz-rec-ru-mow-01 That was just a short outage, sadly that server is not as stable as the other ones, especially when the hardware node or one of the other customers' VPSes gets attacked.
